src/db/tripledb.plt

```diff
@@ -0,0 +1,60 @@
+:- begin_tests('db/tripledb').
+
+:- use_module(library('db/tripledb'),
+    [ tripledb_load/2,
+      tripledb_ask/3,
+      tripledb_forget/3,
+      tripledb_tell/3
+    ]).
+
+% load swrl owl file for tripledb testing
+test('tripledb_load(URL)') :-
+  tripledb_load(
+    'package://knowrob/owl/test/swrl.owl',
+    [ graph(common),
+      namespace(knowrob, 'http://knowrob.org/kb/swrl_test#')
+    ]).
+
+% check via tripledb_ask if individual triple exists
+test('tripledb_ask(S,P,O)') :-
```

Comment:
Maybe better name it:

    test('tripledb_ask(Adult,subclassOf,TestThing)')

then you get more specific information about what was failing in the test.